### ISSUE TITLE
Problem with fancybox

### ISSUE THREAD

**Issue by gapplef** [theme-next/hexo-theme-next#296]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***
I find a problem with fancybox similar to #126:
For images in the body of article, the problem has been fixed.
For images in the front matter, still have the similar problem.

### Expected behavior 
![screenshot from 2018-05-16 20-07-14](https://user-images.githubusercontent.com/5604665/40116380-1a510088-5946-11e8-8573-4d1cbc3bfe85.png)

### Actual behavior 
![screenshot from 2018-05-16 20-08-18](https://user-images.githubusercontent.com/5604665/40116466-55be19bc-5946-11e8-9530-1a2175478171.png)

### Steps to reproduce the behavior 
1. Put image link in the `photo` filed of the `front matter`
2. Turn on fancybox in `_config.yml` of NexT

### Node.js and NPM Information

```
v8.11.1
6.0.1
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.7.1"
  },
  "dependencies": {
    "hexo": "^3.5.0",
    "hexo-deployer-git": "^0.3.1",
    "hexo-generator-archive": "^0.1.4",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-feed": "^1.2.2",
    "hexo-generator-index": "^0.2.0",
    "hexo-generator-searchdb": "^1.0.8",
    "hexo-generator-tag": "^0.2.0",
    "hexo-renderer-ejs": "^0.3.0",
    "hexo-renderer-markdown-it-plus": "^1.0.3",
    "hexo-renderer-stylus": "^0.3.1",
    "hexo-server": "^0.2.2",
    "hexo-tag-aplayer": "^3.0.3"
  }
}
```

#### Hexo version

```
hexo: 3.7.1
```

#### Hexo Configuration

```yml

```

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [x] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [x] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [ ] Gemini

#### NexT Configuration:

```yml
fancybox: true
```

**Comment by stevenjoezhang** [comment]
It's a bug.
https://github.com/theme-next/hexo-theme-next/blob/9810af292a59209a4fc1186f6f3bfc8aaabd7e9a/source/js/src/utils.js#L8-L11
https://github.com/theme-next/hexo-theme-next/blob/9810af292a59209a4fc1186f6f3bfc8aaabd7e9a/layout/_macro/post.swig#L268-L285

**Comment by stevenjoezhang** [comment]
Fixed in #667
